addons/account/static/src/xml/account_reconciliation.xml

```diff
@@ -61,6 +61,7 @@
                     rel="do_action"
                     data-action_name="Unreconciled Entries"
                     data-model="account.move.line"
+                    data-domain="[('parent_state', '=', 'posted')]"
```

Comment:
Maybe add `search_default_posted` in the context so that it isn't "hardcoded"?

Reply:
I don't think it is ok, since the domain is related to the parent state and not the entry itself

Reply:
I'm not sure I see what you mean?
https://github.com/odoo/odoo/blob/c1b9c96dddaa02209bae042481781dd175a678dc/addons/account/views/account_move_views.xml#L327-L328